.github/workflows/build.yml

```diff
@@ -16,9 +16,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Install dependencies
-        run: sudo apt-get -y -q install cmake gcc libeigen3-dev
```

Comment:
Seems like it still uses cmake and gcc? Or are use just assuming the github runner already has those installed?